docs/api/lights/SpotLight.html

```diff
@@ -62,7 +62,7 @@
 
 		<h3>[property:Object3D target]</h3>
 		<div>
-			Spotlight focus points at target.position.<br />
+			Spotlight focus points at target.position. *Note* that currently for target property to work properly, it must be part of the [page:Scene scene].<br />

```

Comment:
Note: If the target position is changed from the default value, the target must be added to the scene graph ( scene.add( light.target ) ).